app/serializers/user_analytics_serializer.rb

```diff
@@ -40,7 +40,12 @@ def core_team_of
 
   def partner_of
     return [] if Rails.env.test?
-    object.partnerships.pluck(:name)
+    partnerships = Product.
+       joins(:transaction_log_entries).
+       where(transaction_log_entries: { wallet_id: object.id }).
+       group('products.id')
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.